diff --git a/Telegram/SourceFiles/boxes/confirm_box.cpp b/Telegram/SourceFiles/boxes/confirm_box.cpp
index 0db838bd1..77c3c314e 100644
--- a/Telegram/SourceFiles/boxes/confirm_box.cpp
+++ b/Telegram/SourceFiles/boxes/confirm_box.cpp
@@ -398,7 +398,7 @@ void PinMessageBox::prepare() {
 	addButton(langFactory(lng_cancel), [this] { closeBox(); });
 
 	if (_peer->isChat() || _peer->isMegagroup()) {
-		_notify.create(this, lang(lng_pinned_notify), true, st::defaultBoxCheckbox);
+		_notify.create(this, lang(lng_pinned_notify), false, st::defaultBoxCheckbox);
 	}
 
 	auto height = st::boxPadding.top() + _text->height() + st::boxPadding.bottom();
